package.json

```diff
@@ -37,10 +38,10 @@
   },
   "dependencies": {
     "body-parser": "^1.18.3",
+    "classnames": "^2.2.6",
```

Comment:
`^` - you should remove these signs
Without them you specify a concrete version of a dependency, which is better, because
if someone introduces bug into the library in the next version your app won't suffer

Reply:
I didn't realise it. `^` has added by npm automatically in every my project. There are other signs also [https://stackoverflow.com/questions/22343224/whats-the-difference-between-tilde-and-caret-in-package-json](url) . On the other hand there is a chance to fix a bugs in library along with the new version released. But I will do as you suggest